File: src/Modules/Contracts/Group.Contract/DTOs/GroupDto.cs
using SharedKernel.Domain;

namespace Group.Contract.DTOs;

public sealed record GroupDto(
      Guid Guid=default,
      string Name="",
      List<GroupDoorDto> Doors=default!, 
      int LocationId=0, 
      bool IsActive=false,
      bool IsDefault=false
      ) : BaseDtoEntity(Guid, LocationId, string.Empty, IsActive,IsDefault);

File: src/Web/src/model/Group/GroupDto.ts
import { BaseDto } from "../BaseDto";
import { GroupDoorDto } from "./GroupDoorDto";

export interface GroupDto extends BaseDto{
    id:number;
    componentId:number;
    name:string;
    doors:GroupDoorDto[];
    locationId:number;
    isActive:boolean;
}

